src/main/java/org/synyx/urlaubsverwaltung/overtime/web/OvertimeViewController.java

```diff
@@ -263,4 +269,10 @@ private void prepareModelForEdit(Model model, Person signedInUser, Person person
         model.addAttribute(SIGNED_IN_USER, signedInUser);
         model.addAttribute(IS_OFFICE_ATTRIBUTE, signedInUser.hasRole(OFFICE));
     }
+
+    private boolean isUserIsAllowedToWriteOvertime(Person signedInUser, Person personOfOvertime) {
```

Comment:
What is the best location for this? 
* `DepartmentService`
* `SettingsService`
* `OvertimeService`
* Stick with duplication: here and in `org.synyx.urlaubsverwaltung.overview.OverviewViewController#isUserIsAllowedToWriteOvertime`

Reply:
Moved to `OvertimeService`